src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.searchBar {
+  position: relative;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 20px 0 0 ;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchBarSmall {
+  position: relative;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 20px 0 0;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
```

Comment:
there are lot of similar properties on `searchBar` & `searchBarSmall ` please make a separate class for that and add those similar properties to that class. and if properties are diffrent have seperate class for that